tests/monotouch-test/MetalPerformanceShaders/KernelTest.cs

```diff
@@ -24,7 +23,11 @@ public class KernelTest {
 		[Test]
 		public void RectNoClip ()
 		{
+#if !MONOMAC
 			TestRuntime.AssertXcodeVersion (7,0);
+#else
+			TestRuntime.AssertXcodeVersion(9, 0);
```

Comment:
Your editor's formatting settings are wrong (there should be a space before the opening parenthesis). This shows up several times below.